lib/rolodex/writers/file_writer.ex

```diff
@@ -1,14 +1,14 @@
 defmodule Rolodex.Writers.FileWriter do
   @behaviour Rolodex.Writer
 
-  alias Rolodex.Config
+  alias Rolodex.{Config, WriterConfig}
 
   @impl Rolodex.Writer
   def init(config) do
-    path = fetch_file_path(config)
-
-    File.touch(path)
-    File.open(path, [:write])
+    with {:ok, path} <- fetch_file_path(config),
+         :ok <- File.touch(path) do
```

Comment:
Right now, unless the user supplies a full valid path in config, this will return `{:error, :enoent}`. Maybe we should add some logic to support dynamic paths based on the project directory?